packages/inspection-capture-web/src/PhotoCapture/hooks/usePhotoCaptureSightState.ts

```diff
@@ -174,9 +191,15 @@ export function usePhotoCaptureSightState({
         try {
           const alreadyTakenSights = getSightsTaken(inspectionId, response);
           setSightsTaken(alreadyTakenSights);
-          setSelectedSight(captureSights.filter((s) => !alreadyTakenSights.includes(s))[0]);
+          selectSight(captureSights.filter((s) => !alreadyTakenSights.includes(s))[0]);
           setLastPictureTaken(getLastPictureTaken(inspectionId, response));
           assertInspectionIsValid(inspectionId, response, captureSights, tasksBySight);
+          analytics.trackEvent('inspection started', {
+            newInspection: !!alreadyTakenSights,
+            alreadyTakenSights: alreadyTakenSights.length,
+            haveToCaptureSights: captureSights.length,
```

Comment:
rename to `remainingSights`

Reply:
Actually, it is the total of sights the user have to take for the whole inspection. `totalSights` is better?